### PR TITLE
Fixes premium menu displaying after setting keys

### LINKED ISSUE #1243: Hide "Get Rotki Premium" when Premium is enabled at the same user session
After entering the correct API key/secret and enabling data sync, the menu bar still displays "Get Rotki Premium".  To a paid-up user this can be a minor annoyance because it feels like superfluous marketing for something which has already been paid.  So ideally that menu item should just be hidden after Premium is activated.

### PR DESCRIPTION
**Pull request by kelsos**
Closes #1243

### CODE REVIEW

**Comment by LefterisJP** [comment]
Failed backend test is due to problems cryptocompare faced yesterday. Should be fine now.

Is this PR ready for a review @kelsos ?

**Comment by kelsos** [comment]
I would like to add some tests too. The plan is do it later today if that is OK.

**Comment by LefterisJP** [comment]
@kelsos definitely. Just ping me when I can review. If it's only tests I can also review if it works.

**Comment by kelsos** [comment]
Sure check the functionality. It should be done.

File: frontend/app/src/components/settings/PremiumSettings.vue [hunk 1]
```diff
@@ -22,6 +22,7 @@
               prepend-icon="fa-key"
               :disabled="premium && !edit"
               :type="showKey ? 'text' : 'password'"
+              :error-messages="errorMessages"
```
Comment (LefterisJP) [inline frontend/app/src/components/settings/PremiumSettings.vue:25]: I made a new user account to test this. What surprised me is that not only the get premium popup was missing. But the first time I logged in also the "Get Premium" at the top menu was missing.



Second time it was there fine.

There was this issue where we wanted to remove the popup but not also the menu entry. That should always be there if the user does not have premium.

Comment (kelsos) [reply frontend/app/src/components/settings/PremiumSettings.vue:25]: Ok give me some time to verify.

Comment (LefterisJP) [reply frontend/app/src/components/settings/PremiumSettings.vue:25]: It's fixed.

**Pull request merged**